crates/emath/src/rect.rs

```diff
@@ -191,6 +191,18 @@ impl Rect {
         Self::from_min_max(self.min - amnt, self.max + amnt)
     }
 
+    /// Scale up by this factor in each direction, keeping the center
+    #[must_use]
+    pub fn scale(self, amnt: f32) -> Self {
+        self.scale2(Vec2::splat(amnt))
+    }
+
+    /// Scale up by this factor in each direction, keeping the center
+    #[must_use]
+    pub fn scale2(self, scale: Vec2) -> Self {
```

Comment:
I think these should be called `scale_from_center[2]`, or take an `Align2` to specify which corner is the fixpoint

Reply:
Renamed to `scale_from_center`. I don't think taking an `Align2` would be particularly useful (I can't think of a use case for wanting to scale up around non-center?), but I might be wrong on that.